components/src/preact/mutationFilter/mutation-filter.tsx

```diff
@@ -24,55 +25,63 @@ const mutationFilterPropsSchema = mutationFilterInnerPropsSchema.extend({
 export type MutationFilterInnerProps = z.infer<typeof mutationFilterInnerPropsSchema>;
 export type MutationFilterProps = z.infer<typeof mutationFilterPropsSchema>;
 
-export type SelectedFilters = {
-    nucleotideMutations: (SubstitutionClass | DeletionClass)[];
-    aminoAcidMutations: (SubstitutionClass | DeletionClass)[];
-    nucleotideInsertions: InsertionClass[];
-    aminoAcidInsertions: InsertionClass[];
+type SelectedNucleotideMutation = {
+    type: 'nucleotideMutations';
+    value: SubstitutionClass | DeletionClass;
 };
 
+type SelectedAminoAcidMutation = {
+    type: 'aminoAcidMutations';
+    value: SubstitutionClass | DeletionClass;
+};
+
+type SelectedNucleotideInsertion = {
+    type: 'nucleotideInsertions';
+    value: InsertionClass;
+};
+
+type SelectedAminoAcidInsertion = {
+    type: 'aminoAcidInsertions';
+    value: InsertionClass;
+};
+
+export type MutationFilterItem =
+    | SelectedNucleotideMutation
+    | SelectedAminoAcidMutation
+    | SelectedNucleotideInsertion
+    | SelectedAminoAcidInsertion;
+
 export const MutationFilter: FunctionComponent<MutationFilterProps> = (props) => {
     const { width, initialValue } = props;
     return (
         <ErrorBoundary
-            size={{ height: '3.375rem', width }}
+            size={{ height: '40px', width }}
             layout='horizontal'
             schema={mutationFilterPropsSchema}
             componentProps={props}
         >
             <div style={width}>
```

Comment:
Huh, that already didn't work before.
```suggestion
            <div style={{width}}>
```